examples/knative/python/kn-py-slack/function.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-py-slack
+  labels:
+    app: veba-ui
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+spec:
+  template:
+    metadata:
+    spec:
+      containers:
+        - image: projects.registry.vmware.com/veba/kn-py-slack:1.0
+          envFrom:
+            - secretRef:
+                name: slack-secret
+          env:
+            - name: FUNCTION_DEBUG
```

Comment:
The secretRef.name.slack-secret is used because the handler code picks up the secret that is created earlier in the process by the same name.  Lines 21 and 22 are not used in my example.  I was following the template in the other Slack example.